src/perform_step/ssprk_perform_step.jl

```diff
@@ -514,7 +516,8 @@ end
   step_limiter!(u, f, t+dt)
 
   if integrator.opts.adaptive
-    @. atmp = (utilde-u)/(integrator.opts.abstol+max(integrator.opts.internalnorm(uprev),integrator.opts.internalnorm(u))*integrator.opts.reltol)
+    @. utilde = utilde - u
```

Comment:
Is it possible to transform the coefficients such that we only need to perform one linear combination, but not to subtract two arrays? e.g. https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/perform_step/low_order_rk_perform_step.jl#L23

Reply:
I don't see how this could be possible without using more memory to save more temporary arrays.

Reply:
Ah, I see. Thanks for explaining!